### DISCUSSION TITLE
Is that possible to jump back from response title?

### DISCUSSION THREAD

**Discussion by cortrano** [nathanhoad/godot_dialogue_manager#532] [Q&A]
For example i have this dialogue file.

```
~ start
Hello GitHub!
=>< question
The end of the dialogue
=> END 

~ question
Do you like to contribute to opensource?
- Yes!
   Cool!
- No!
   That's sad(

```

But when I choose one of the options, dialogue don't jump back me to the "start" title. Dialogue just closes.

**Comment by nathanhoad** [comment]
Looks like it's the missing explicit `=> END` at the end of the file. I'll change the compiler to automatically append one.
